tools/dep_updaters/update-histogram.sh

```diff
@@ -0,0 +1,74 @@
+#!/bin/sh
+set -e
+# Shell script to update histogram in the source tree to specific version
+
+BASE_DIR=$(cd "$(dirname "$0")/../.." && pwd)
+DEPS_DIR="$BASE_DIR/deps"
+
+[ -z "$NODE" ] && NODE="$BASE_DIR/out/Release/node"
+[ -x "$NODE" ] || NODE=$(command -v node)
+
+# shellcheck disable=SC1091
+. "$BASE_DIR/tools/dep_updaters/utils.sh"
+
+NEW_VERSION="$("$NODE" --input-type=module <<'EOF'
+const res = await fetch('https://api.github.com/repos/HdrHistogram/HdrHistogram_c/releases/latest');
+if (!res.ok) throw new Error(`FetchError: ${res.status} ${res.statusText}`, { cause: res });
+const { tag_name } = await res.json();
+console.log(tag_name.replace('v', ''));
+EOF
+)"
+
+CURRENT_VERSION=$(grep "#define HDR_HISTOGRAM_VERSION" ./deps/histogram/include/hdr/hdr_histogram_version.h | sed -n "s/^.*VERSION \"\(.*\)\"/\1/p")
+
+echo "Comparing $NEW_VERSION with $CURRENT_VERSION"
+
+if [ "$NEW_VERSION" = "$CURRENT_VERSION" ]; then
+  echo "Skipped because histogram is on the latest version."
+  exit 0
+fi
+
+echo "Making temporary workspace"
+
+WORKSPACE=$(mktemp -d 2> /dev/null || mktemp -d -t 'tmp')
+
+cleanup () {
+  EXIT_CODE=$?
+  [ -d "$WORKSPACE" ] && rm -rf "$WORKSPACE"
+  exit $EXIT_CODE
+}
+
+trap cleanup INT TERM EXIT
+
+HISTOGRAM_TARBALL="$NEW_VERSION.tar.gz"
```

Comment:
IIRC the other scripts use tarball names of this form:

```suggestion
HISTOGRAM_TARBALL="histogram-v$NEW_VERSION.tar.gz"
```

Reply:
This needs to match the file name published to GH?
e.g. from https://github.com/HdrHistogram/HdrHistogram_c/releases/tag/0.11.8 this should match https://github.com/HdrHistogram/HdrHistogram_c/archive/refs/tags/0.11.8.tar.gz

Reply:
@richardlau exactly it needs to match

Reply:
Oh but only because the variable appears in the URL. In any case, I don't really care.